src/tests/system/tests/test_failover.py

```diff
@@ -80,3 +80,30 @@ def test_failover__reactivation_timeout_is_honored(
     assert (
         f"Primary server reactivation timeout set to {expected} seconds" in log
     ), f"'Primary server reactivation timeout set to {expected} seconds' not found in logs!"
+
+
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopology.IPA)
+def test_failover__connect_second_family(client: Client, ipa: IPA):
+    """
+    :title: Make sure that we can connect using secondary protocol
+    :setup:
+        1. Create user in IPA
+        2. Set family_order to "ipv6_first"
+        3. Set IPv6 address in /etc/hosts so it resolves but it
+           points to non-exesting machine
+    :steps:
+        1. Restart SSSD
```

Comment:
Technically, you are starting SSSD. This line can be the 4th step in the setup, so the test only contains one step.

Reply:
done